app/assets/stylesheets/layouts/_grid.scss

```diff
@@ -8,10 +8,14 @@
 }
 
 .l-grid {
-  border-top:  1px solid $border-color-1;
+  border-top: 1px solid $border-color-1;
 
   .row {
     .-no-mrg { margin: 0; }
+
+    .-br-bottom {
```

Comment:
Is this a state? it seems that it is a child instead of a state. We should only use '.-' when it's a state or modificator of the current selector